### DISCUSSION TITLE
pricing - Buildorder and Purchase Orders

### DISCUSSION THREAD

**Discussion by mabroens** [inventree/InvenTree#4796] [Q&A] [answered]
Hi,
I have been reading a lot about pricing and PO/SO/BO but unfortunately still left with some questions.

In my case i have a PCBA that need to be build so the part has a BOM with components that i provide to the  assembly house. 
But the assembly house will also purchase several parts, so these are not part of my bom but instead i created a part called **AssyCost_EMS-A_Partxxx**.  
![image](https://github.com/inventree/InvenTree/assets/130482028/2e06bff9-d236-4012-a33c-edec51c093e2)

Depending on the qty i have them assemble i get different prices so these are added in the price break tab
![image](https://github.com/inventree/InvenTree/assets/130482028/eb1ec485-0039-4463-b860-9674ac34edcf)

**Q1:** is adding the production cost to the BOM the way to do this? or is there a better way?? 

When i look at the pricing tab for the part itself then you can see more information.
![image](https://github.com/inventree/InvenTree/assets/130482028/c151ce84-df7b-4682-993e-f74c02a48124)

and then there's a field called supplier pricing:
![image](https://github.com/inventree/InvenTree/assets/130482028/70ac2e46-2a32-4009-9ba3-4348e308634a)
or should i have entered these cost here. in the end i see them as BOM cost since that plus the cost for the parts i provide make up the cost for a PCB.

Okay so now i have a sold 20 pcb's. so i create a **SO**, but i have no stock so i also create a **BO** 
**Q2:** how are these linked??

**Q3:** do i also need to create a PO? in the end i want the pricing info to be part of the part pricing.

How do others use this?
I'd like to hear what the idea behind this was, im a doing it right and how do others use/see this

Thanks in advance
Martijn

**Comment by SchrodingersGat** [comment, marked as answer]
This sounds like it might relate to an open issue - https://github.com/inventree/InvenTree/issues/1071

**Reply by mabroens** [reply]
Hi @SchrodingersGat yes that covers it i think.

**Reply by SchrodingersGat** [reply]
@mabroens thanks - currently there is not much attention to that issue but it is still something we plan on adding support for
